x/gov/simulation/genesis_test.go

```diff
@@ -40,15 +40,16 @@ func TestRandomizedGenState(t *testing.T) {
 	var govGenesis types.GenesisState
 	simState.Cdc.MustUnmarshalJSON(simState.GenState[types.ModuleName], &govGenesis)
 
-	dec1, _ := sdk.NewDecFromStr("0.400000000000000000")
-	dec2, _ := sdk.NewDecFromStr("0.458000000000000000")
-	dec3, _ := sdk.NewDecFromStr("0.528000000000000000")
-	dec4, _ := sdk.NewDecFromStr("0.324000000000000000")
+	dec1, _ := sdk.NewDecFromStr("0.375000000000000000")
+	dec2, _ := sdk.NewDecFromStr("0.487000000000000000")
+	dec3, _ := sdk.NewDecFromStr("0.524000000000000000")
+	dec4, _ := sdk.NewDecFromStr("0.313000000000000000")
 
-	require.Equal(t, "905stake", govGenesis.DepositParams.MinDeposit.String())
-	require.Equal(t, "77h26m10s", govGenesis.DepositParams.MaxDepositPeriod.String())
-	require.Equal(t, float64(317894), govGenesis.VotingParams.VotingPeriod.Seconds())
-	require.Equal(t, float64(107823), govGenesis.VotingParams.ExpeditedVotingPeriod.Seconds())
+	require.Equal(t, "272stake", govGenesis.DepositParams.MinDeposit.String())
+	require.Equal(t, "41h11m36s", govGenesis.DepositParams.MaxDepositPeriod.String())
+	require.Equal(t, "800stake", govGenesis.DepositParams.MinExpeditedDeposit.String())
+	require.Equal(t, float64(270511), govGenesis.VotingParams.VotingPeriod.Seconds())
+	require.Equal(t, float64(137225), govGenesis.VotingParams.ExpeditedVotingPeriod.Seconds())
```

Comment:
nit: how / where are we getting these numbers?

Reply:
These are defined randomly. When we add new fields to the params, many related values change because the order of the randomized initialization is now different. I normally just make sure that only expected values get changed and copy over the new values